### ISSUE TITLE
Cannot install kfserving SDK from setup.py

### ISSUE THREAD

**Issue by jinchihe** [kubeflow/kfserving#426]
/kind bug

**What steps did you take and what happened:**
```
(python3) hejinchidembp:kfserving hejinchi$ python setup.py install
running install
running bdist_egg
running egg_info
writing kfserving.egg-info/PKG-INFO
......
Installed /Users/hejinchi/PycharmProjects/python3/lib/python3.6/site-packages/kfserving-0.2.0-py3.6.egg
Processing dependencies for kfserving==0.2.0
error: azure-storage-blob 12.0.0b3 is installed but azure-storage-blob~=1.3 is required by {'azure'}
(python3) hejinchidembp:kfserving hejinchi$
```

**What did you expect to happen:**

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Istio Version:
- Knative Version:
- KFServing Version:
- Kubeflow version:
- Minikube version:
- Kubernetes version: (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

**Comment by jinchihe** [comment]
That's OK if comment out the `azure-storage-blob >= 2.1.0` from https://github.com/kubeflow/kfserving/blob/282def3efc0bf99d852600f71fa4ff90cd08923f/python/kfserving/requirements.txt#L11

**Comment by jinchihe** [comment]
In fact, we can remove this, because azure will also install the `azure-storage-blob`

**Comment by jinchihe** [comment]
Already fixed in #383.
/close
